roles/splunk/tasks/configure_license.yml

```diff
@@ -1,4 +1,33 @@
 ---
+- block:
+  - name: Copy Licenses to Master
```

Comment:
Please change to `  - name: Copy licenses to master` to be consistent with the rest of the role